build-setup/macos.md

```diff
@@ -53,6 +53,7 @@ The first major piece of development software to install is Apple's development
 - If you are using Mojave, you must install Xcode 10.3.x
```

Comment:
We should mention Catalina now, and we also know that Xcode 11 is fine and works on Mojave.  So please change this to:

- If you are using Mojave or Catalina, you must install Xcode 11.2.x